docker/start.py

```diff
@@ -205,36 +205,43 @@ def main(args, environ):
             config_dir, config_path, environ, ownership
         )
 
-    if mode is not None:
-        error("Unknown execution mode '%s'" % (mode,))
+    args = ["python"] + args[1:]
 
-    config_dir = environ.get("SYNAPSE_CONFIG_DIR", "/data")
-    config_path = environ.get("SYNAPSE_CONFIG_PATH", config_dir + "/homeserver.yaml")
+    if "-m" not in args:
+        args = ["-m", synapse_worker] + args
```

Comment:
you're putting ` ["-m", synapse_worker]` *before* `python`?

Reply:
indeed, fixed that